tfm_secure_partitions/tfm_huk_deriv_srv/tfm_huk_deriv_srv.h

```diff
@@ -34,6 +34,8 @@
 #define LABEL_LO    LABEL_CONCAT(_EC_PRIV_KEY_LO)
 #define LABEL_UUID  LABEL_CONCAT(UUID)
 
+#define LABEL_BUFF_SIZE 40
```

Comment:
This should probably be defined in the .c file, since it's never used outside that source module. There's no point exposing it to everyone that includes this header file.